### PR TITLE
Deployment pipeline to firebase app distribution

### PR DESCRIPTION
**Pull request by aravind-raveendran**

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -4,6 +4,51 @@ before_all do
...
+  gym(
+    workspace: ENV['GYM_WORKSPACE'],
+    scheme: ENV['GYM_SCHEME'],
+    clean: true,
+    output_directory: ENV['GYM_OUTPUT_DIR'],
+    configuration: ENV['GYM_CONFIGURATION'],
+    build_path: ENV['GYM_BUILD_PATH'],
+    destination: ENV['GYM_DESTINATION'],
+    export_method: ENV['GYM_EXPORT_METHOD'],
+    xcargs: "MARKETING_VERSION='1.1.#{build_number}' CURRENT_PROJECT_VERSION='#{build_number}'",
```
Comment (CasperMok) [inline fastlane/Fastfile:39]: Do all the versions have to be 1.1.x in the future?

Comment (aravind-raveendran) [reply fastlane/Fastfile:39]: Hi Casper, We currently don't have a proper versioning mechanism for our apps. We followed the practice of picking up the build number from the GitHub Action build jobs and appending it to the static Major and Minor app versions. So currently if we have to do a Major or Minor app version change, it has to be done here..

Comment (CasperMok) [reply fastlane/Fastfile:39]: Please commit it here, so we are more aware of these necessary changes in the future.

Comment (aravind-raveendran) [reply fastlane/Fastfile]: Done !!

**Pull request merged**